### PR TITLE
new fingerprint and sample config for log4j

### PR DESCRIPTION
**Pull request by andrwils**
I added a new fingerprint to handle the log4j app logs. I also added some sample config for testing the change.

### CODE REVIEW

**Review by DerekRoberts — APPROVED** [review]

File: event-stream-processing/src/constants/fingerprints.ts [hunk 1]
```diff
@@ -166,6 +166,35 @@ export const FINGERPRINTS: OsDocumentFingerprint[] = [
       },
     },
   },
+  {
+    name: FingerprintCategory.TOMCAT_LOG4J_LOGS,
+    fingerprint: {
+      event: {
+        kind: 'event',
+        category: 'web',
```
Comment (mbystedt) [inline event-stream-processing/src/constants/fingerprints.ts:174]: Category should be an array.

https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-category

**Review by mbystedt — CHANGES_REQUESTED** [review]

File: event-stream-processing/src/constants/fingerprints.ts (hunk 1 above)
Comment (andrwils) [reply event-stream-processing/src/constants/fingerprints.ts:174]: Such as category: ['web']?

Comment (andrwils) [reply event-stream-processing/src/constants/fingerprints.ts]: @mbystedt I updated all the event categories accordingly.

**Comment by andrwils** [comment]
I'm closing this and will open a new one once the other changes have been merged.

**Pull request closed without merging**